Modules/TOF/src/TOFMatchedTracks.cxx

```diff
@@ -122,9 +156,17 @@ void TOFMatchedTracks::monitorData(o2::framework::ProcessingContext& ctx)
     }
     // loop over TOF MatchInfo
     for (const auto& matchTOF : mTPCTOFMatches) {
-      const auto& trk = mTPCTracks[matchTOF.getTrackRef().getIndex()];
+      GTrackID gTrackId = matchTOF.getTrackRef();
+      const auto& trk = mTPCTracks[gTrackId.getIndex()];
       mMatchedTracksPt[trkType::UNCONS]->Fill(trk.getPt());
       mMatchedTracksEta[trkType::UNCONS]->Fill(trk.getEta());
+      if (mUseMC) {
+	auto lbl = mRecoCont.getTrackMCLabel(gTrackId);
```

Comment:
Hi @chiarazampolli indeed here probably the formatting CI  fails

Reply:
Yes, I also noticed, sorry, I pushed now the fix.